.github/workflows/alphabetise.yml

```diff
@@ -0,0 +1,53 @@
+name: Alphabetise extension list
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - antora-playbook.yml
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install packages
+        run: npm install -g yaml-sort
```

Comment:
Where is this package used?

Reply:
Ugh, good spot! That was leftover from an earlier attempt which used javascript (unwisely, as it turns out).